### DISCUSSION TITLE
Is there a way to have "packages"?

### DISCUSSION THREAD

**Discussion by sgorblex** [twpayne/chezmoi#2196] [Q&A] [answered]
Something I like about the `stow` dotfiles workflow is that you can bundle your dotfiles in groups of files, independent from the directory they are in. For example, I may have a "neovim" directory (package) which, when stowed, links files in `~/.config/nvim` as well as a file somewhere else read by the shell which sets the `EDITOR` env variable (the shell may have its own package, but this file does not belong to that, even if it configures the shell somehow).
Is there any clean way to accomplish this with chezmoi? I might be missing something but it looks to me like you can only apply or remove everything (unless excluded by some template).

**Comment by twpayne** [comment, marked as answer]
In chezmoi machine-to-machine differences within a single file are managed with templates. For including or excluding whole sets of files, use `.chezmoiignore`.

In your case you would use `.chezmoiignore` to include or exclude the `~/.config/nvim` directory and make your `.bashrc` a template to set the `EDITOR` environment variable.
